tools/run_tests/python_utils/start_port_server.py

```diff
@@ -111,15 +112,15 @@ def start_port_server(port_server_port):
       try:
         urllib.request.urlopen('http://localhost:%d/get' % port_server_port,
                         timeout=1).read()
-        print('port server is up and ready')
+        logging.info('port server is up and ready')
         break
       except socket.timeout:
-        print('waiting for port_server: timeout')
+        logging.exception('waiting for port_server: timeout')
         traceback.print_exc();
```

Comment:
I think this is no longer necessary; I think the call to `logging.exception` prints this information to the log.